File: applications/node/src/chat-completions/hello.ts
import client, { MODEL } from "@/client";

export async function sayHello() {
  const chatCompletion = await client.chat.completions.create({
    messages: [
      { role: "user", content: "Please repeat this and only this. Hello Test" },
    ],
    model: MODEL,
  });

  return chatCompletion.choices[0].message.content;
}

export async function sayHelloTwice() {
  const chatCompletion = await client.chat.completions.create({
    messages: [
      { role: "user", content: "Please repeat this and only this. Hello Test" },
    ],
    model: MODEL,
    n: 2,
  });

  return chatCompletion.choices.map(({ message }) => message.content);
}

File: applications/python/chat_completion/hello.py
from client import client
from config import MODEL

def hello():
    response = client.chat.completions.create(
        messages=[
            {
                "role": "user",
                "content": "Please repeat this and only this. Hello Test",
            }
        ],
        model=MODEL,
    )

    return response.choices[0].message.content
